examples/starfield/main.rs

```diff
@@ -69,19 +71,21 @@ impl Starfield {
 
 #[derive(Clone, Copy)]
 struct Star {
-    a: f32,
-    b: f32,
+    color: (u8, u8, u8),
+    local_x: f32,
+    local_y: f32,
     x: f32,
-    z: f32,
+    size: f32,
 }
 
 impl Star {
     fn new_rand(w: usize, h: usize) -> Star {
         let mut star = Star {
-            a: 0.0,
+            local_x: 0.0,
             x: 0.0,
-            b: 0.0,
-            z: 0.0,
+            local_y: 0.0,
```

Comment:
Personally I prefer the a and b variable names as they are also used in the formula y = ax + b. I admit this was not clear in the original code. Maybe we can group the a and b variables together and add y = ax + b as a comment.

Reply:
Ah, that makes sense - I can update it and leave a comment or something

Reply:
Are you alright with z -> size? I'm realizing now that it was probably named that way to make you think of 3d space.

Reply:
Yes size is fine as it is not really 3D anyway.